src/Domain/MatchFundsRemainingService.php

```diff
@@ -27,9 +27,33 @@ class MatchFundsRemainingService
      * confusing and I'd rather avoid.
      *
      * So it looks like we're not ready to create an implementation better than the below just yet.
+     *
+     * @todo change return from float to numeric-numeric string for precision and to match how we deal with money
+     * across matchbot.
      */
-    public function getFundsRemaining(CampaignDomainModel $campaign): float
+    public function getFundsRemaining(CampaignDomainModel $campaign, ?MetaCampaign $metaCampaign): float
     {
```

Comment:
below is a direct PHP port of the formula field from SF. But as mentioned in commit message I'm thinking now porting is probably the wrong approach for this.